tomcat/src/main/java/org/apache/coyote/common/response/Response.java

```diff
@@ -0,0 +1,89 @@
+package org.apache.coyote.common.response;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.coyote.common.Charset;
+import org.apache.coyote.common.Header;
+import org.apache.coyote.common.HttpVersion;
+import org.apache.coyote.common.MediaType;
+
+public class Response {
+
+    private static final String RESPONSE_MESSAGE_DELIMITER = "\r\n";
+
+    private final String startLine;
+    private final String headers;
+    private final String body;
+
+    private Response(final String startLine, final String headers, final String body) {
+        this.startLine = startLine + " ";
+        this.headers = headers;
+        this.body = body;
+    }
+
+    public String getResponse() {
+        return String.join(RESPONSE_MESSAGE_DELIMITER,
+                startLine,
+                headers,
+                body);
+    }
+
+    public static class ResponseBuilder {
+
+        private static final String HEADER_KEY_VALUE_DELIMITER = ": ";
+        private static final String HEADER_BODY_DELIMITER = "";
+        private static final String HEADER_VALUE_DELIMITER = ";";
+
+        private final String startLine;
+        private final Map<String, String> headers;
+        private String body;
+
+        public ResponseBuilder(final HttpVersion httpVersion, final Status status) {
+            this.startLine = String.join(" ", httpVersion.getValue(), status.getValue());
+            this.headers = new HashMap<>();
+            headers.put(Header.CONTENT_TYPE.getValue(), MediaType.TEXT_HTML.getValue());
+            this.body = "";
+        }
+
+        public ResponseBuilder setContentType(final MediaType mediaType, final Charset charset) {
+            return setHeader(Header.CONTENT_TYPE,
+                    combineHeaderValues(mediaType.getValue(), "charset=" + charset.getValue()));
+        }
+
+        private String combineHeaderValues(final String... values) {
+            return String.join(HEADER_VALUE_DELIMITER, values);
+        }
+
+        public ResponseBuilder setContentType(final MediaType mediaType) {
+            return setContentType(mediaType, Charset.UTF8);
+        }
+
+        public ResponseBuilder setContentLength(final int length) {
+            return setHeader(Header.CONTENT_LENGTH, length + "");
```

Comment:
```suggestion
            return setHeader(Header.CONTENT_LENGTH, length);
```
빈문자열을 더해준 이유가 있을까요? 👀

Reply:
setHeader의 두 번째 파라미터는 String 타입이지만 length는 int여서 캐스팅 해준거에요